.github/workflows/pr-labeler.yml

```diff
@@ -1,6 +1,6 @@
 name: PR Labeler
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
We don't need both?

Reply:
Nope, apparently, `pull_request_target` will work for both.

Reply:
However, it seems that it is not triggering on this PR anymore.